haystack/components/rankers/transformers_similarity.py

```diff
@@ -40,6 +40,8 @@ def __init__(
         device: str = "cpu",
         token: Union[bool, str, None] = None,
         top_k: int = 10,
+        metadata_fields_to_embed: Optional[List[str]] = None,
```

Comment:
I think we should change it to `meta_fields_to_embed`, all dataclasses are standardizing to `meta` instead of `metadata`.

Reply:
Agree, even if we have other components using `metadata`. I opened an issue for that: https://github.com/deepset-ai/haystack/issues/6570

Reply:
changed!